index.js

```diff
@@ -286,8 +285,18 @@ class Raspi extends EventEmitter {
   }
 
   pinMode(pin, mode) {
+    this._pinMode({ pin, mode });
+  }
+
+  _pinMode(options) {
```

Comment:
Style nit: can we make this a Symbol instead of an underscore? That way, the method is truly private. To do this, add a line on line 68 that looks like:

```
const pinMode = Symbol('pinMode');
```

Then, change this line to look like:

```
[pinMode](pin, mode) {
```

and finally replace all calls to `this._pinMode` with `this[pinMode]`.